torch_xla/csrc/unwrap_data.cpp

```diff
@@ -9,14 +8,13 @@
 
 #include "absl/types/span.h"
 #include "torch_xla/csrc/runtime/computation_client.h"
-#include "torch_xla/csrc/xla_data.h"
 
 namespace torch_xla {
 
 runtime::ComputationClient::DataPtr UnwrapXlaData(
```

Comment:
I actually would rename this function to something like `DynamicCastToXlaDatas` and other one to `DynamicCastToLazyDatas`, since it is not really a wrapping. We can do that in a follow up pr through.

Reply:
That name makes more sense. I wonder if we can/should add a general utility to dynamic-cast a vector of shared pointers. I'll see if there's any other usage.